flatpak/build_icons.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+curl https://www.mamedev.org/MAMELogoR-trans.png -o logo.png
+for i in 128; do
+    convert logo.png -resize ${i}x${i} -gravity center -background none -extent ${i}x${i} mame${i}x${i}.png
+done
```

Comment:
Yes, the SVG would be preferred.  As far as I know there is no provision for a smaller M style logo.  The only real requirement is for a 128x128.  I did notice while using MAME on the SteamDeck there is a need for a small square logo where the M logo would be a good fit.  Not sure where it would fit in this package though.  Also it seems KDE hover on the task bar can show a bigger logo, so maybe a 256x256 is nice to have.  Haven't tested the 256x256 logo yet though.

Reply:
Here’s the MAME icon rendered at various typical “icon” sizes.
16\*16 ![mameicon16x16](https://user-images.githubusercontent.com/9678431/183255460-7b531185-2307-4973-8c1a-9ab0fa9ab01a.png)
32\*32 ![mameicon32x32](https://user-images.githubusercontent.com/9678431/183255462-51d32c9f-2423-4f40-a580-b4217e18da85.png)
128\*128 ![mameicon128x128](https://user-images.githubusercontent.com/9678431/183255464-5412909a-7bba-42f8-8e3e-d61bf6de48a3.png)
256\*256 ![mameicon256x256](https://user-images.githubusercontent.com/9678431/183255467-d92e4045-819b-4b09-abb4-3e5891892c66.png)
SVG suitable for “large” sizes ![mameicon](https://user-images.githubusercontent.com/9678431/183255469-9b3b50d5-d48d-4a3c-ba45-be0d58522db5.svg)

Reply:
On reflection, I think I like the 32\*32 icon better with sharp mitres
![mameicon32x32](https://user-images.githubusercontent.com/9678431/183255588-0d205769-e258-4374-9f9e-f162519be1ba.png)

Reply:
And just because I can’t leave things well alone, here are the sizes all regenerated.  Source has been updated to use this style for the Windows icon.
![mameicon16x16](https://user-images.githubusercontent.com/9678431/183258492-e9358db1-7368-4040-a993-4ce3e3b75329.png)
![mameicon24x24](https://user-images.githubusercontent.com/9678431/183258496-bda6097f-2a8b-42c9-9a4e-f1c096f2baab.png)
![mameicon32x32](https://user-images.githubusercontent.com/9678431/183258497-a18fa2f3-61c0-478f-ba03-b781daea8a85.png)
![mameicon48x48](https://user-images.githubusercontent.com/9678431/183258499-112fbada-cf93-4d1b-97d7-fd6b77749ec1.png)
![mameicon64x64](https://user-images.githubusercontent.com/9678431/183258502-dd6db18d-9bd2-4d68-a3be-b0e476744f5d.png)
![mameicon128x128](https://user-images.githubusercontent.com/9678431/183258504-f4f4136a-c065-46e6-bf70-27682d4f13d5.png)
![mameicon256x256](https://user-images.githubusercontent.com/9678431/183258507-8888c6c8-ffc4-46ef-a422-70631952dabc.png)